bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,121 @@
+work with any internet-connected weather station, that gets it's dns server via DHCP or where the DNS server can be
+set. 
+
+## Discovery
+
+In the initial version there is no discovery, but it is planned for a later release. This will also automatically 
+generate channels base on the update request.  
+In this version you need to manually add a thing and configure the station id, and only create items for channels the
+weather station submits values for. E.g. the above-mentioned Fine Offset-based stations doesn't submit values for the averages over time.
+These values can be calculated in rules if desirable and even appended to the wunderground submission if the other measurements
+are forwarded to wunderground.com in a rule.
+
+## Thing Configuration
+
+The only configurable value is the station id, which should match the one configured on the weather station. If you don't plan on submitting
+measurements to wunderground.com, it can be any unique non-empty string value.
+
+## Channels
+
+Each measurement type the wunderground.com update service accepts has a channel. The channels are named teh same as the request parameters they receive.
+Additionally there is a receipt timestamp and a trigger channel.
+
+##### State channels:
+The names match the possible parameters as documented at https://support.weather.com/s/article/PWS-Upload-Protocol?language=en_US
+in addition to some undocumented ones that the WH2650 weather station does include.
+   
+| channel          | type                 | description                                                                                        |
+|------------------|----------------------|----------------------------------------------------------------------------------------------------|
+| lastReceived     | DateTime             | The date and time of the last update.                                                              |
+| dateutc          | String               | The date and time of the last update in UTC as submitted by the weather station. This can be 'now' |
+| softwaretype     | String               | A software type string from the weather station                                                    |
+| lastQueryState   | String               | The part of the last query after the first unurlencoded ?                                          |   
+| windspeedmph     | Number:Speed         | Current wind speed, using software specific time period                                            |
+| winddir          | Number:Angle         | Current wind direction angle                                                                       |
+| windgustmph      | Number:Speed         | Current wind gust speed                                                                            |
+| windgustdir      | Number:Angle         | Wind gust direction angle                                                                          |
```

Comment:
All channels should be in lower camelCase. For example windSpeed and windDirection. Since you are using UOM dont include the unit in the channel ID.